app/controllers/catalog_controller.rb

```diff
@@ -104,6 +104,7 @@ def enforce_show_permissions(_opts = {})
         lc_subject_label_tesim
         location_label_tesim
         title_tesim
+        all_text_timv
```

Comment:
Is `all_text` the conventional index name, rather than `full_text`?

Reply:
Yeah, that's the field that's used in Hyrax for example